bait-files/src/main/webapp/submission.js

```diff
@@ -0,0 +1,209 @@
+function loadAmericanAddress() {
```

Comment:
I'm a little confused - the only differences between the addresses are postal code vs zip code and province vs state right? Couldn't the other inputs just be set up using HTML? Would keep things DRY - otherwise the address and city code here could probably go into one separate function to simplify these two functions.

Reply:
Yes, but the main problem with that is the Select element.  Each of those options would need to change which is done in the javascript + I thought there would be two different types of objects so each label + input pair has a name and for pairing.  I can make a function to make new labels + text input fields so that it is more DRY!

Reply:
Changed to make more dry